README.md

```diff
@@ -135,11 +136,9 @@ Started from **Mar 13**
 
 #### **Round 3: Competitive Individual Race**
 
-Participants receive NFT airdrops on IRISnet and perform Interchain NFT Transfer through specified flows in Task Data. The airdrop has two rounds: In the first round, all participants go through the same flow, and in the second round, all participants go through different flows.
+In this round, participants will receive NFT airdrops on IRISnet. Each NFT contains `Task Data` in its token data, which includes a `flow` field with a corresponding `flow-id` value that can be used to query the actual flow [in flow table](https://github.com/game-of-nfts/gon-testnets/blob/main/doc/flow-table.md). Participants can then perform Interchain NFT Transfer using the specified flows. After completing the interchain NFT transfer, each participant must transfer that NFT to the `last_owner` and then complete the race.
```

Comment:
The last part: "and then complete the race.", do you actually mean that there is anything left to do after transfering to `last_owner`, or is transferring to `last_owner` completing the race?

Reply:
Transfer to `last_owner` is the last thing you need to do. Just keep your tx hash for evidence.

Reply:
@taramakage does it mean that an IBC transfer to `las_owner` is invalid?

Reply:
> @taramakage does it mean that an IBC transfer to `las_owner` is invalid? i need to use TransferNFT '

Thanks for noticing this! Do IBC transfer to your iris address first, then transfer that NFT to the last owner. For example:

```
IBC transfer:  iris-addr -> juno-addr -> stars-addr -> uptick-addr -> iris-addr (your iris-addr receives that interchain nft)

transfer ownership on iris:  iris-addr -> last-owner (finish this step then complete the race)
```

Reply:
> > @taramakage does it mean that an IBC transfer to `las_owner` is invalid? i need to use TransferNFT '
> 
> Thanks for noticing this! Do IBC transfer to your iris address first, then transfer that NFT to the last owner. For example:
> 
> ```
> IBC transfer:  iris-addr -> juno-addr -> stars-addr -> uptick-addr -> iris-addr (your iris-addr receives that interchain nft)
> 
> transfer ownership on iris:  iris-addr -> last-owner (finish this step then complete the race)
> ```

Ok, thanks.